src/App.tsx

```diff
@@ -49,6 +52,22 @@ if (process.env.REACT_APP_ENVIRONMENT !== 'production') {
 type Props = {};
 
 function App(props: Props) {
+  window.addEventListener('storage', event => {
```

Comment:
Should this be in an `useEffect`? That way you could remove the event listener on dismount.

Reply:
When it comes to `localStorage` it seems that value changes won't trigger, not even inside `useEffect`.  This solution was only way that I managed to achieve automatic functionality.